GTG/plugins/send_email/sendEmail.py

```diff
@@ -32,22 +33,38 @@ class pluginSendEmail:
     """
     The plugin.
     """
+    def __init__(self):
+        """
+        Initialize variables.
+        """
+        self.browser_plugin_api = None
+        self.taskview_plugin_api = None
+        self.tb_Managerbutton = None
+        self.tb_Taskbutton = None
```

Comment:
You need to keep track of all buttons that you added, not just one. Also rename it to conform pep8:

```python
self.editor_buttons = []
```